locales/de.json

```diff
@@ -0,0 +1,96 @@
+{
+	"File": "Datei",
+	"Quit": "Schliessen",
+	"Edit": "Bearbeiten",
+	"Undo": "Lösen",
+	"Redo": "Noch einmal",
+	"Cut": "Schneiden",
+	"Copy": "Kopieren",
+	"Paste": "Einfügen",
+	"Delete": "Löschen",
+	"Select all": "Alles markieren //?",
```

Comment:
`Alles auswählen`
Btw. where did `//?` come from?

Reply:
The // ? indicated that i was unsure of the translation. This is a menu entry.

Reply:
Ah okay. 😅 That makes sense :)